libs/domain/order/totals/totals.component.ts

```diff
@@ -0,0 +1,25 @@
+import { TotalsController } from '@spryker-oryx/cart';
+import { ContentMixin } from '@spryker-oryx/experience';
+import { hydratable, i18n, signal, signalAware } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+
+@hydratable('window:load')
+@signalAware()
+export class OrderTotalsComponent extends ContentMixin(LitElement) {
+  protected totalsController = new TotalsController(this);
+
+  connectedCallback(): void {
+    super.connectedCallback();
+
+    this.totalsController.provideContext('ORDER');
+  }
+
+  protected $totals = signal(this.totalsController.getTotals());
```

Comment:
alternative:

```ts

  protected $totals = computed(() => {
    this.totalsController.provideContext('ORDER');
    return this.totalsController.getTotals();
  });
```

Reply:
Not sure. On each change in totals it will re-provide the context. Better to keep them separate